devnotes/pnor_layout.md

```diff
@@ -0,0 +1,181 @@
+| SBKT                        | 0x1D | 0x03d02000:0x03D05FFF  | 0x4000       | d    | -     | same as size |                                       |
+| [HDAT](#HDAT)               | 0x1E | 0x03d06000:0x03D0DFFF  | 0x8000       | d    | -     | same as size |                                       |
+| UVISOR                      | 0x1F | 0x03d10000:0x03E0FFFF  | 0x100000     | d    | -     | same as size |                                       |
+| [BOOTKERNFW](#BOOTKERNFW)   | 0x20 | 0x03e10000:0x03FEFFFF  | 0x1E0000     | d    | -     | same as size |                                       |
+| [BACKUP_PART](#BACKUP_PART) | 0x21 | 0x03ff7000:0x03FFEFFF  | 0x8000       | d    | -     | 0            |                                       |
+
+Type:
+* l = FFS_TYPE_LOGICAL
+* d = FFS_TYPE_DATA
+* p = FFS_TYPE_PARTITION
+
+Flags:
+* b = FFS_FLAGS_U_BOOT_ENV
+* p = FFS_FLAGS_PROTECTED
+
+# Reproducibility
+Reproducibility of partitions can be found at the
+[wiki page](https://wiki.raptorcs.com/wiki/Firmware/Reproducible_Builds/Status).
+
+# Partition original usage
+## part
+This partition contains partition table.
+
+## HBEL
+This partition is used by `hostboot` to to save crash information for later
+debug.
+More informations can be found at the
+[wiki page](https://wiki.raptorcs.com/wiki/Hostboot_Debug_Howto).
+
+## GUARD
+This partition is responsible for holding information about components that
+are detected as broken.
+More information about it can be found at the
+[wiki page](https://wiki.raptorcs.com/wiki/Troubleshooting/Guard_Partition).
+
+## NVRAM
+Non volatile memory required for OPAL systems.
+More informations can be fouind on the
+[wiki page](https://open-power.github.io/skiboot/doc/opal-api/opal-nvram-read-write-7-8.html).
+
+## SECBOOT
+This partition is used to hold secure variables across renppts in
+tamper-resistant manner. Because writes to PNOR can't be prevented, hash of
+this partition is stored in the `TPM NV`, and verified at the boot time.
+More informations can be found at the
+[wiki page](https://open-power.github.io/skiboot/doc/secvar/secboot_tpm.html).
+
+## HBB
+Bootloader code for `hostboot`
+
+## HBD
+Hostboot data
+
+## HBI
+Main `hostboot` application
+
+## SBE
+Self Boot Engine, stored in module EEPROM
+
+## HCODE
+FW for PPE
+
+## PAYLOAD
+This partition contains `skiboot`.
+
+## BOOTKERNEL
+This partition contains `skiroot`, `petitboot` and initramfs.
+
+## OCC
+Most likely contains FW for OCC (On Chip Controller).
+Source code for this FW is available [here](https://git.raptorcs.com/git/talos-occ/).
+
+## VERSION
+Probably just an information about versions of a binaries in the PNOR image.
+
+## HBBL
+This partition is used by hostboot during update operation.
+
+## IMA_CATALOG
+This partition holds a list of events supported by OCC microcode.
+
+## MEMD
+Holds VPD for RAM IIRC.
```

Comment:
`IIRC` ? Let's make sure what's there

Reply:
I'm not sure, where this information can be found. For now I used description from pnor layout xml https://github.com/open-power/pnor/blob/master/p9Layouts/defaultPnorLayout_64.xml